HomeWork1/src/test/java/hw1/DivisionTest.java

```diff
@@ -0,0 +1,11 @@
+package hw1;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DivisionTest extends DataProviderTestClass {
+    @Test(dataProvider = "calculator DataProvider with 2 double args")
+    public void divMethodTest(double a, double b){
+        Assert.assertEquals(calculator.div(a, b), a / b);
```

Comment:
It is better to use `static import` for the `assertions`. Please check all test classes